src/Shaders/ShadersInclude/instancesDeclaration.fx

```diff
@@ -3,6 +3,7 @@
 	attribute vec4 world1;
 	attribute vec4 world2;
 	attribute vec4 world3;
+    attribute vec4 instanceColor;

```

Comment:
I think it should be enclosed inside `#ifdef VERTEXCOLOR` ... `#endif`?

Reply:
I've added a new define for instance color because it's possible to have instances with mesh vertex color but no instance Color.